src/translators/errors.py

```diff
@@ -51,11 +62,18 @@ def __init__(self, error: Exception):
     def error(self) -> Exception:
         return self._error
 
-    def is_transient_error(self) -> bool:
+    def is_aggregation_error(self) -> str:
+        e = self._error
+        if isinstance(e, pg8000.ProgrammingError):
+            if len(e.args) > 0 and isinstance(e.args[0], dict) \
+                and e.args[0].get('C', '') == '42883':
+                return ("AggrMethod cannot be applied")
+
+    def is_transient_error(self):
         e = self._error
         if isinstance(e, struct.error):                    # (1)
             msg = str(e)
-            return msg.startswith('unpack_from requires a buffer')
+            return ('unpack_from requires a buffer')
```

Comment:
please revert to `msg.startswith('unpack_from requires a buffer')`

Reply:
reverted